src/plugins/links/public/components/editor/links_panel_editor_empty_prompt.tsx

```diff
@@ -10,28 +10,22 @@ import React from 'react';
 
 import { EuiText, EuiPanel, EuiSpacer, EuiButton, EuiEmptyPrompt, EuiFormRow } from '@elastic/eui';
 
-import { NavEmbeddableStrings } from '../navigation_embeddable_strings';
+import { LinksStrings } from '../links_strings';
 
-export const NavigationEmbeddablePanelEditorEmptyPrompt = ({
-  addLink,
-}: {
-  addLink: () => Promise<void>;
-}) => {
+export const LinksPanelEditorEmptyPrompt = ({ addLink }: { addLink: () => Promise<void> }) => {
```

Comment:
```suggestion
export const LinksEditorEmptyPrompt = ({ addLink }: { addLink: () => Promise<void> }) => {
```

+ rename file to `links_editor_empty_prompt.tsx`